### ISSUE TITLE
Maintain 7.1 Audio

### ISSUE THREAD

**Issue by drsassafras** [mbunkus/mkvtoolnix#2114]
I've found that when processing 7.1 Audio (DTS-ES) It drops one channel in the copy process leaving only 6 channels of audio (instead of the 7 that it should be) Or perhaps it just does not read the file correctly after stripping the other audio tracks?

Media-Info says there are 7 tracks, but when I open the channels for editing, only 6 show up. So I'm pretty sure it is being stripped but does not change the metadata (which of course says it has the original 7 channels)

The use-case is in removing a 5.1 audio channel and leaving only a 7.1 channel. Both being English.

**Comment by mbunkus** [comment]
Please upload a sample file before muxing to my [FTP server](https://github.com/mbunkus/mkvtoolnix/wiki/FTP-server). Thanks.

**Comment by mbunkus** [comment]
> but when I open the channels for editing

How exactly do you do that?

**Comment by drsassafras** [comment]
I opened the file in audacity to view the channels.

So an update. It appears as though what I reported earlier was incorrect. The original source is 6.1, so it has 6 channels. Now, when I load that source into MkvTool it shows the source as having 7 channels. Media-Info reports the 5 channel as having 6 and the 6 channel audio as having 7. As does  MkvTool, but the HandBrake and the listed audio formats (5.1 & 6.1) both agree that there are 5 & 6 channels.

So I'm not sure if this is a bug, but having different software report conflicting channels is confusing.

Let me know if you still want that sample.
